### PR TITLE
busayo's profile

### PR DESCRIPTION
**Pull request by amoweolubusayo**
Kindly help review my PR 😃

### CODE REVIEW

File: content/profiles/busayo-amowe.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Olubusayo Amowe"
+profileimage: "/images/profile-pictures/busayo-amowe.jpg"
+coverimage: "/images/profile-pictures/busayo-amowe-cover.jpeg"
+tags: ["CSharp", ".NET",".NET core","Azure","Devops","AKS"]
+twitter: "amoweo"
+github: "amoweolubusayo"
+linkedin : "https://www.linkedin.com/in/amoweolubusayo"
+comments: false
+summary: "I am a .NET Developer and a huge man of MS products."
```
Comment (Layla-P) [inline content/profiles/busayo-amowe.md:10]: IS this a typo? Should it be huge "fan"?

**Review by Layla-P — CHANGES_REQUESTED** [review]
Thank you Olubusayo!

I think there is just a little typo :-)

File: content/profiles/busayo-amowe.md (hunk 1 above)
Comment (amoweolubusayo) [reply content/profiles/busayo-amowe.md:10]: My bad lol. I must have been thinking of a man. Will correct it now. Thanks for noticing this

Comment (amoweolubusayo) [reply content/profiles/busayo-amowe.md]: @Layla-P fixed the typo

**Comment by amoweolubusayo** [comment]
> Thank you Olubusayo!
> 
> I think there is just a little typo :-)

Typo fixed now. Thank you.

**Pull request merged**